lib/rubocop/cop/style/redundant_regexp_character_class.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for unnecessary single-element Regexp character classes.
+      #
+      # @example
+      #
+      #   # bad
+      #   r = /[x]/
+      #
+      #   # good
+      #   r = /x/
+      #
+      #   # bad
+      #   r = /[\s]/
+      #
+      #   # good
+      #   r = /\s/
+      #
+      #   # good
+      #   r = /[ab]/
+      class RedundantRegexpCharacterClass < Cop
+        include MatchRange
+
+        MSG_REDUNDANT_CHARACTER_CLASS =
+          'Redundant single-element character class, `[` and `]` can be removed'
```

Comment:
You forgot the final `.`. I'd also make this more actionable and word it like `Redundant ... [x] can be replaced with x.` (or something along those lines).